d2/src/main/java/com/linkedin/d2/balancer/zkfs/LastSeenLoadBalancerWithFacilities.java

```diff
@@ -50,7 +51,8 @@
  *
  * @author Francesco Capponi (fcapponi@linkedin.com)
  */
-public class LastSeenLoadBalancerWithFacilities implements LoadBalancerWithFacilities {
+public class LastSeenLoadBalancerWithFacilities implements LoadBalancerWithFacilities, WarmUpService
```

Comment:
Given that WarmUpService interface is supported everywhere, can we move this interface into LoadBalancer (or merge them) and have an impl in LoadBalancerWithFacilitiesDelegator to save a lot of code?

Reply:
Great suggestion. Can we do this refactoring as part of a different pull request so that we can unblock the LastSeen rollout quickly?